docs/sources/upgrade-guide/upgrade-v8.0/index.md

```diff
@@ -0,0 +1,64 @@
+---
+aliases:
```

Comment:
I think we should remove the empty aliases line.
```suggestion
```

Same for other upgrade guides.